ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/clientlibs/site/js/textinputview.js

```diff
@@ -120,6 +122,46 @@
 
         _updateValue(value) {
             this.widget.value = value;
+            this._runValidations(this.widget);
+        }
+
+        _runValidations(widget) {
+            const model = this._model._jsonModel;
+            let validationObj = {isError: false, errorMessage: ''};
+            this._runRequiredValidation(widget, model, validationObj);
```

Comment:
view should not run validations. model should run them. view should only be responsible for setting value in model.